### PR TITLE
Giving ProductStreams some love

### PR DESCRIPTION
**Pull request by pascalheidmann-bedarf**
### 1. Why is this change necessary?
Convenience.

### 2. What does this change do, exactly?
Nothing ground breaking just using the vast amount of whitespace in productstream selection grid to display the article number (useful if you work with a lot of similar named products) as well the ability to directly open the article itself.

### 6. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: themes/Backend/ExtJs/backend/product_stream/view/search_grid.js [hunk 1]
```diff
@@ -112,7 +112,21 @@ Ext.define('Shopware.apps.ProductStream.view.SearchGrid', {
     createActionItems: function() {
         var me = this;
         return [{
+            iconCls: 'sprite-inbox',
+            action: 'openArticle',
```
Comment (shyim) [inline themes/Backend/ExtJs/backend/product_stream/view/search_grid.js:116]: This line is useless 😅

File: themes/Backend/ExtJs/backend/product_stream/view/search_grid.js [hunk 2]
```diff
@@ -112,7 +112,21 @@ Ext.define('Shopware.apps.ProductStream.view.SearchGrid', {
...
+                    name: 'Shopware.apps.Article',
+                    action: 'detail',
+                    params: {
+                        articleId: record.get('articleId')
+                    }
+                });
+            }
+        }, {
             iconCls: 'sprite-minus-circle-frame',
+            tooltip: '{s name=removeProduct}Remove article from stream{/s}',
```
Comment (mitelg) [inline themes/Backend/ExtJs/backend/product_stream/view/search_grid.js:129]: not the same text as in the snippet file

File: themes/Backend/ExtJs/backend/product_stream/view/search_grid.js (hunk 1 above)
Comment (mitelg) [reply themes/Backend/ExtJs/backend/product_stream/view/search_grid.js:116]: or name it "openProduct"

**Comment by pascalheidmann-bedarf** [comment]
@shyim @mitelg good points.
I will fix those as soon as I get the time and will also take a look on the similar designed filtered productstream but I won't make it before next week  - so please don't close this PR because of perceived inactivity ;)

**Pull request closed without merging**